lms/tasks/roster.py

```diff
@@ -65,6 +65,8 @@ def schedule_fetching_course_rosters() -> None:
                     no_recent_roster_clause,
                     recent_launches_cluase,
                 )
+                # Prefer newer courses
```

Comment:
We plan to start fetching roster more frequently now: 

https://github.com/hypothesis/lms/pull/6660

With this change we'll try to get one roster for most recent course/assignments and then work in the backlog.